### PR TITLE
Liota integration with kura/ESF

### PR DESCRIPTION
**Pull request by lihaoyucool**
The main function of this package is to integrate kura and liota.
Kura is configured to send json packets to liota. Liota automatically creates Edge Systems and metrics based on the json format.
Just put it in the userpackage directory and it will execute automatically.

### CODE REVIEW

File: examples/integration_with_kura.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+import logging
+import json
+import time
+import threading
+import random
+import copy
+from liota.entities.devices.simulated_device import SimulatedDevice
+from liota.entities.metrics.metric import Metric
+from liota.device_comms.mqtt_device_comms import MqttDeviceComms
+from liota.core.package_manager import LiotaPackage
```
Comment (KohliDev) [inline examples/integration_with_kura.py]: This is a package will reside in "liota/packages/examples"

File: examples/integration_with_kura.py [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+class PackageClass(LiotaPackage):
+
+    BrokerIP = "127.0.0.1"
+    BrokerPort = 1883
+    BrokerTopic = "#"
+    metrics = []
+    reg_devices = []
+
+    """Subscibe callback function"""
+    def sub_callback(self, client, userdata, msg):
```
Comment (KohliDev) [inline examples/integration_with_kura.py]: Keep it outside the PackageClass

File: examples/integration_with_kura.py [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+        if (msg.topic.split("/").__len__() == 4) and (msg.topic.split("/")[3] == "pulse-topic"):
+
+            Jsonpayload = json.loads(msg.payload)['metrics']
+            if Jsonpayload:
+
+                #is new gateway?
+                if Jsonpayload['assetName'] in Gatewaydict.keys():
```
Comment (KohliDev) [inline examples/integration_with_kura.py]: Please add comments on the need of the need of json parsing

File: examples/integration_with_kura.py [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+        """
+        The execution function of a liota package.
+
+        Acquires "iotcc_mqtt" and "iotcc_mqtt_edge_system" from registry and registers edge_system related metrics
+        with the DCC and publishes those metrics.
+
+        :param registry: the instance of ResourceRegistryPerPackage of the package
+        :return:
+        """
+        self.iotcc = registry.get("iotcc_mqtt")
```
Comment (KohliDev) [inline examples/integration_with_kura.py]: Please add detailed comments on how data is getting published to IoTCC after json Parsing.

**Comment by lihaoyucool** [comment]
@KohliDev Thankl you! I already addressed your comments in new pull request so just close this.

**Pull request closed without merging**